pages/index.tsx

```diff
@@ -1,9 +1,5 @@
 export default function Home() {
-	return (
-		<>
-			<h1>Dashboard</h1>
-		</>
-	);
+	return <></>;
```

Comment:
I think this component is unused and if it's used it's rendering anything

Reply:
I think that it has to return something, otherwise I'm getting errors :shrug: